src/LayerPlan.cpp

```diff
@@ -1104,7 +1105,7 @@ void LayerPlan::addLinesByOptimizer
         }
         boundary.add(comb_boundary_minimum.offset(dist));
         // simplify boundary to cut down processing time
-        boundary.simplify(MM2INT(0.1), MM2INT(0.1));
+        boundary = Simplify(MM2INT(0.1), MM2INT(0.1), 0).polygon(boundary);
```

Comment:
I know these where originally pretty arbitrary, but now the parameters aren't the squares of those values anymore, which is a big change even so.

Reply:
No, this was part of the confusion of the original function, really.

`PolygonRef::simplify` indeed takes squared distances.
`Polygons::simplify` does NOT take squared distances, but squares the value itself before passing it to `PolygonRef::simplify` for each of its sub-polygons.